example-app/SantokuApp/src/features/qa-question/components/TagSheet.tsx

```diff
@@ -1,43 +1,26 @@
 import {m} from 'bases/message/Message';
 import {BottomSheet} from 'bases/ui/bottom-sheet/BottomSheet';
-import {StyledTouchableOpacity, Text} from 'bases/ui/common';
-import {StyledColumn} from 'bases/ui/common/StyledColumn';
+import {Box, StyledTouchableOpacity, Text} from 'bases/ui/common';
 import {StyledRow} from 'bases/ui/common/StyledRow';
 import {StyledSpace} from 'bases/ui/common/StyledSpace';
 import {TagIllustration} from 'bases/ui/illustration/TagIllustration';
 import {Tag as TagType} from 'features/backend/apis/model';
-import React, {useCallback, useState} from 'react';
+import React from 'react';
 
 import {Tag} from './Tag';
 
 type TagSheetProps = {
   tags?: TagType[];
   isVisible: boolean;
-  initialSelectedTagIds: string[];
-  select: (tagIds: string[]) => void;
+  clear: () => void;
+  decide: () => void;
+  selectTag(tagId: string): void;
+  isActive(tagId: string): boolean;
 };
-export const TagsSheet: React.FC<TagSheetProps> = ({tags, isVisible, initialSelectedTagIds, select}) => {
-  const [selectedTagIds, setSelectedTagIds] = useState(initialSelectedTagIds);
-  const selectTag = useCallback(
-    (value: string) => {
-      if (selectedTagIds.includes(value)) {
-        setSelectedTagIds(v => v.filter(i => i !== value));
-      } else {
-        setSelectedTagIds(v => [...v, value]);
-      }
-    },
-    [selectedTagIds],
-  );
-
-  const clear = useCallback(() => setSelectedTagIds([]), []);
-
-  const selectTags = useCallback(() => select(selectedTagIds), [select, selectedTagIds]);
-  const isActive = useCallback((tagId: string) => selectedTagIds.includes(tagId), [selectedTagIds]);
-
+export const TagSheet: React.FC<TagSheetProps> = ({tags, isVisible, clear, decide, selectTag, isActive}) => {
   return (
-    // TODO: BottomSheetは暫定的な実装です。
-    <BottomSheet isVisible={isVisible} containerStyle={{backgroundColor: 'rgba(0.5, 0.25, 0, 0.2)'}}>
-      <StyledColumn p="p16" bg="white" borderTopLeftRadius="p16" borderTopRightRadius="p16">
+    <BottomSheet isVisible={isVisible}>
```

Comment:
I want the bottom tab to close when I click BackDrop.

Reply:
Fixed to close on back drop tap
29e90ed3e8562452c8b28412a26028b1cdda76f5

Reply:
Please do the same for SingleSelectableSortSheet.

Reply:
Fixed.
123d2445c7d08d2f226c56a80b062492d200b61c